source/triggers/database-triggers.txt

```diff
@@ -643,6 +643,10 @@ the trigger to fire. For details, see the reference for each event type:
 - :manual:`replace </reference/change-events/replace>`
 - :manual:`delete </reference/change-events/delete>`
 
+We recommend only using match expressions when the volume of Trigger events
```

Comment:
should this be in a callout? esp since the heading is telling users to use match expressions and this recommendation to limit its use seems easy to skip over

Reply:
I think people skip note blocks so I just left it inline. I was tempted to move it to the beginning of the section though.